.github/workflows/run_tests.yaml

```diff
@@ -13,39 +13,16 @@ jobs:
       - name: Install requirement
         run: python -m pip install -r requirements.txt
       - name: Tool extraction
-        # run: bash bin/extract_all_tools.sh
         run: | 
-          python bin/extract_galaxy_tools.py \
-            extractools \
-            --api $GITHUB_API_KEY \
-            --all-tools "results/test_tools.tsv" \
-            --all-tools-json "results/test_tools.json" \
-            --planemo-repository-list "test.list" \
-            --test
+          bash bin/extract_all_tools.sh test 
```

Comment:
why did u switch back to the bash script, does `set -e` work to make it stop on python error ?